test/stripe/util_test.rb

```diff
@@ -33,39 +33,11 @@ class UtilTest < Test::Unit::TestCase
         g: [],
       }
       assert_equal(
-        "a=3&b=%2Bfoo%3F&c=bar%26baz&d[a]=a&d[b]=b&e[]=0&e[]=1&f=",
+        "a=3&b=%2Bfoo%3F&c=bar%26baz&d[a]=a&d[b]=b&e[0]=0&e[1]=1&f=",
```

Comment:
Do we have a test that verifies an array of hashes too? `{foo: [{bar: :a}, {bar: :b}]}` -> `foo[0][bar]=a&foo[1][bar]=b`?

Reply:
Yeah! Check just a bit lower in `"#url_encode should prepare strings for HTTP"`, and it seems to do a pretty good job.